scss/mixins.scss

```diff
@@ -0,0 +1,3 @@
+@mixin font($font) {
+  font-family: $font;
+}
```

Comment:
add empty line and define for all text this prop in fonts.scss

Reply:
Done

Reply:
:arrow_up: